### DISCUSSION TITLE
Visitor.

### DISCUSSION THREAD

**Discussion by Pikaronga** [BG-Software-LLC/SuperiorSkyblock2#426] [Pending Feature Requests] [answered]
Add a way to disable the visitors location and use the island home instead. a toggle in the config file. 
use-visitor-sign: false 
would make the /is visit (name) command tp to the island home instead. This would be a lot easier for players to understand and a nice feature. (most likely not that hard to implement)

**Comment by Lemvs** [comment, marked as answer]
Implemented for a long time.

@OmerBenGera you can mark this answered and close it.
